src/EventGridExtension/EventGridExtension.csproj

```diff
@@ -1,9 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
+    <TargetFramework>net46</TargetFramework>
```

Comment:
hmm, why this change?

Reply:
since now I no longer have a one nuget package targeting both frameworks, I am following the conventions to separate version and dependencies:
![image](https://user-images.githubusercontent.com/6531400/38280253-32d6efd2-3759-11e8-8ab1-88f8eeb9f7d3.png)
![image](https://user-images.githubusercontent.com/6531400/38280256-3e8478e0-3759-11e8-8c55-9b4fb163411d.png)